solutions/swb-reference/integration-tests/support/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+//Average start waiting time is 5 minutes, setting start max waiting to 8 minutes
```

Comment:
As we add more environment types, their wait times will differ and we will need to come back here to change this. To simplify things for now, could we just have one timeout for all operations:

`export const RESOURCE_POLL_MAX_WAIT_SECONDS: number = 480;`

Reply:
Also add a common polling interval constant to be used by polling operations.